### DISCUSSION TITLE
[Documentation] Notice that a single root element is required now missing from docs?

### DISCUSSION THREAD

**Discussion by kokoshneta** [livewire/livewire#7187] [7. Bugs (archived)]
### Livewire version

v3

### Steps To Reproduce

In [the Quickstart section of the v2 docs](https://laravel-livewire.com/docs/2.x/quickstart#create-a-component), there was a notice that Livewire MUST have a single root element.

In the [corresponding page of the v3 docs](https://livewire.laravel.com/docs/quickstart), this notice appears to have been removed – despite the fact that Livewire components must still have exactly one single root element. As far as I’ve been able to ascertain through searching and reading, there is no mention of this requirement anywhere at all in the current documentation.

While Livewire will now throw an exception if multiple root elements are detected, the notice is still very useful for users, particularly new users, and should not have been removed.

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@kokoshneta it was probably just an oversight, PRs welcome to update the docs 🙂

**Reply by kokoshneta** [reply]
@joshhanley I’d be happy to make a PR for the documentation. If I do, though, I’d also like to add which restrictions apply to the root element – for example, as mentioned in #6452, using the `<body>` element – or even a `<section>` – won’t work. Does it actually have to be a `<div>` element specifically?

**Reply by joshhanley** [reply]
@kokoshneta there shouldn't be any restrictions (except maybe `<body>` as using that doesn't make much sense), so they're likely to just be bugs. I would still PR the warning and if there are other issues, it can be updated later to match.

**Reply by kokoshneta** [reply]
PR #7203 created! I also added a related section on the _Components_ page that slots that should go into the containing layout file (outside the component’s root element) – as the only thing – are allowed outside the root element in the Blade view without triggering a ‘multiple root elements detected’ error.

**Reply by patriktoth67** [reply]
Using `<section>`,` <main>` or anything other than `<div>` WITH lazy loading messes up everything in the component (works fine without lazy loading). I've been trying to solve it in a PR, but really I have no idea where to start. If anyone points me in the right direction I'd be happy to look into it.

**Reply by joshhanley** [reply]
@patriktoth67 try setting a placeholder and ensuring it also has the same element and see if that works.

**Reply by patriktoth67** [reply]
You are right, that way it works perfectly. Thank you!

**Reply by joshhanley** [reply]
No worries! 🙂
